test-butler-app/src/main/java/com/linkedin/android/testbutler/GsmDataDisabler.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.android.testbutler;
+
+import android.content.ContextWrapper;
+import android.os.Build;
+import android.os.RemoteException;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.telephony.TelephonyManager;
+import android.util.Log;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * A helper class which can enable or disable data transferring
+ */
+/* package */ final class GsmDataDisabler {
```

Comment:
Yes, but it shows that default access level is used not accidentally.
Nevertheless deleted